base/changes.txt

```diff
@@ -74,7 +74,7 @@ are not part of the distribution.
 
 2021-12-10  David Carlisle  <David.Carlisle@latex-project.org>
 
-	* utf8ienc.dtx: Add inputenc UTF-8 support for for Digraphs U_01C4-U+01CC
+	* utf8ienc.dtx: Add inputenc support for Digraphs U+01C4-U+01CC
```

Comment:
inputenc only handles UTF-8 (and calls it that) so I think it is correct to say UTF-8  here and below. Unicode is broader, if you give pdftex input in some other unicode flavor it will die.